_static/kendrasearchtools.js

```diff
@@ -0,0 +1,690 @@
+/*
+ * searchtools.js
```

Comment:
It would be good to reference the original file directly, and comment about how this one is different.